src/psyclone/psyir/backend/sir.py

```diff
@@ -317,6 +314,10 @@ def binaryoperation_node(self, node):
         operator to SIR.
 
         '''
+        intrinsic_operators = [
```

Comment:
Please add a comment saying what this list is for and again for `binary_operators` below.

Reply:
Added comment.